docker/puppetserver/docker-entrypoint.d/30-set-permissions.sh

```diff
@@ -2,3 +2,5 @@
 
 chown -R puppet:puppet /etc/puppetlabs/puppet/
 chown -R puppet:puppet /opt/puppetlabs/server/data/puppetserver/
+chown -R puppet:puppet /etc/puppetlabs/puppetserver/
+chown -R puppet:puppet /var/log/puppetlabs/puppetserver/
```

Comment:
I think this makes sense... also FYI @jpartlow

Reply:
We may not have had to do this on the pe-puppetserver side because we were much more intentional about how to manage state there. We never came back to the puppetserver container to make it consistent with everything else, so it's a bit of the wild west here.

Reply:
Yeah, looks like in the pe-puppetserver Dockerfile we explicitly chown /etc/puppetlabs and /opt/puppetlabs/server/data (which is where the logs end up in that image).